src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -1259,14 +1265,36 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
    * @returns {Number} - start time of DVR window.
    * @public
    */
-  getStartTimeOfDvrWindow(): number {
+  _getStartTimeOfDvrWindow(): number {
     if (this.isLive() && this._videoElement.seekable.length) {
       return this._videoElement.seekable.start(0);
     } else {
       return 0;
     }
   }
 
+  getStartTimeOfDvrWindow() {
+    return this._startTimeOfDvrWindow;
+  }
+
+  _setInterval() {
+    const intervalTime = 250;
+    this._startTimeOfDvrWindowInterval = setInterval(() => {
+      //get Segment duration
+      const duration = this._segmentDuration;
+      if (
+        !this._waitingEventTriggered && //not in wait
+        this._getStartTimeOfDvrWindow() && //value is not Zero
+        duration && //Duration exist
+        Math.abs(this._getStartTimeOfDvrWindow() - this._startTimeOfDvrWindow) <= duration * 2 //Gap is less than twice the duration
```

Comment:
why specifically this formula ?

Reply:
Since there is jitter between the timer and the duration, gap up to twice the duration reduce the chance to jumps